integration_tests/tests/consistency/consistency_transaction_details.sql

```diff
@@ -0,0 +1,55 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+-- this test ensures the transaction details end model matches the prior version
+with prod as (
+    select *
+    from {{ target.schema }}_netsuite_prod.netsuite2__transaction_details
+    where date(transaction_date) < date({{ dbt.current_timestamp() }})
+),
+
+dev as (
+    select * 
+    --remove before merging
+    except(location_id)
```

Comment:
Reminder to remove before merging so we can take advantage of this going forward.